src/lp/empty_row.jl

```diff
@@ -1,21 +1,71 @@
-# TODO: this is redundant with forcing constraint checks
-#   => an empty row is automatically redundant or infeasible
+@doc raw"""
+    RemoveEmptyRow <: AbstractRule
 
+Eliminate a single row if it is empty, i.e. all variable coefficients are zero.
+In particular, consider row $i$ given as
+```math
+l^{r}_{i} ≤ \sum_{j} a_{i, j} x_{j} ≤ u^{r}_{i}.
+```
+We eliminate row $i$ if, for each variable index $j$,
+``math
+| a_{i,j} | ≤ ϵ
+```
+for prescribed tolerance $ϵ$.
```

Comment:
I don't see this being done in the code, is it on purpose?

Reply:
Hmm no, I wrote this first and then never followed through on it. Do we want to do this? If so, at what point? It feels a bit weird to do it in ``load_problem!``: if the user is passing us the matrix with small nonzeros, it seems like we should respect that.

Reply:
If we do it, I would rather put it in a different rule, with a drop tolerance for rounding small values to zero and large ones to infinity.
As for the where, once at the beginning of presolve seems reasonable. See also https://github.com/mtanneau/MathOptPresolve.jl/issues/12#issuecomment-751526797